build/testing/Clusterfile

```diff
@@ -3,7 +3,7 @@ kind: Cluster
 metadata:
   name: my-cluster
 spec:
-  image: seadent/my-kuberentes-cluster-with-dashboard:latest
+  image: sealer/my-kuberentes-cluster-with-dashboard:latest
```

Comment:
the ACR sealer namespaces is not belong to us. using sealer-io instead

Reply:
ok